content/html/reference/elements/base/docs.md

```diff
@@ -26,7 +26,5 @@ relative URLs contained within a document. There can be only one
 The base URL of a document can be queried from a script using
 [`document.baseURI`](/en-US/docs/Web/API/Document/baseURI).
 
-## Usage notes
-
 If multiple `<base>` elements are specified, only the first `href` and
 first `target` value are used; all others are ignored.
```

Comment:
According to https://github.com/mdn/stumptown-content/issues/89#issue-474656624, this page is also missing "See also".